nexus/test-utils/src/resource_helpers.rs

```diff
@@ -581,6 +585,46 @@ where
     object_create_error(client, &url, body, status).await
 }
 
+pub async fn create_affinity_group(
+    client: &ClientTestContext,
+    project_name: &str,
+    group_name: &str,
+) -> AffinityGroup {
+    object_create(
+        &client,
+        format!("/v1/affinity-groups?project={}", &project_name).as_str(),
+        &params::AffinityGroupCreate {
+            identity: IdentityMetadataCreateParams {
+                name: group_name.parse().unwrap(),
+                description: String::from("affinity group description"),
+            },
+            policy: AffinityPolicy::Fail,
+            failure_domain: FailureDomain::Sled,
+        },
+    )
+    .await
+}
+
+pub async fn create_anti_affinity_group(
+    client: &ClientTestContext,
+    project_name: &str,
+    group_name: &str,
+) -> AntiAffinityGroup {
+    object_create(
+        &client,
+        format!("/v1/anti-affinity-groups?project={}", &project_name).as_str(),
+        &params::AntiAffinityGroupCreate {
+            identity: IdentityMetadataCreateParams {
+                name: group_name.parse().unwrap(),
+                description: String::from("affinity group description"),
```

Comment:
not important, but if you're going to have a description...

Reply:
Updated